meson.build

```diff
@@ -0,0 +1,28 @@
+project('diffuse',
+    version: '0.6.0',
+    meson_version: '>= 0.50',
+    license: 'GPL2+')
+
+i18n = import('i18n')
+python = import('python')
+
+py_installation = python.find_installation('python3')
+
+find_program('gtk-update-icon-cache', required: false)
```

Comment:
What is this for? You don't save the result of it, so you don't seem to actually use it on your meson.build. You don't have to do this for invoking this in the post-install script

Reply:
I have seen this in [Authenticator's `meson.build`](https://gitlab.gnome.org/World/Authenticator/-/blob/master/meson.build). It seems like a good practice, to ensure the program exists. It is not flagged `required` because it is called only in some condition. Again, I copied [Authenticator's `src/meson.build`](https://gitlab.gnome.org/World/Authenticator/-/blob/master/build-aux/meson_post_install.py), it seemed like a good practice.

Reply:
I think noticing the potential use of this program to the user/packager is a good thing. I'll keep it for now.